cli/command/context/create.go

```diff
@@ -107,6 +108,9 @@ func createNewContext(o *CreateOptions, stackOrchestrator command.Orchestrator,
 	if dockerTLS != nil {
 		contextTLSData.Endpoints[docker.DockerEndpoint] = *dockerTLS
 	}
+	if len(o.Kubernetes) != 0 {
+		logrus.Warn("kubernetes orchestrator is deprecated")
```

Comment:
I wanted to suggest to perhaps just use `fmt.Printf()` here (as logrus formatting is a bit "meh" on the CLI), but I'm ok with doing in a follow-up; 

```bash
docker context create --docker host=tcp://localhost:2375 --kubernetes host=http://foo:1234 bla
WARN[0000] kubernetes orchestrator is deprecated
bla
Successfully created context "bla"
```

Perhaps we should generate the warning earlier, or use `flags.MarkDeprecated()` (which both hides the flag, and prints a deprecation message)

Giving it a quick try what it looks like with `flags.MarkDeprecated()`;

<details>

```patch
diff --git a/cli/command/context/update.go b/cli/command/context/update.go
index 55f775037..bcf1a6101 100644
--- a/cli/command/context/update.go
+++ b/cli/command/context/update.go
@@ -3,7 +3,6 @@ package context
 import (
        "bytes"
        "fmt"
-       "github.com/sirupsen/logrus"
        "text/tabwriter"

        "github.com/docker/cli/cli"
@@ -60,8 +59,8 @@ func newUpdateCommand(dockerCli command.Cli) *cobra.Command {
        flags.SetAnnotation("default-stack-orchestrator", "deprecated", nil)
        flags.StringToStringVar(&opts.Docker, "docker", nil, "set the docker endpoint")
        flags.StringToStringVar(&opts.Kubernetes, "kubernetes", nil, "set the kubernetes endpoint")
+       flags.MarkDeprecated("kubernetes", "option will be ignored")
        flags.SetAnnotation("kubernetes", "kubernetes", nil)
-       flags.SetAnnotation("kubernetes", "deprecated", nil)
        return cmd
 }

@@ -102,9 +101,6 @@ func RunUpdate(cli command.Cli, o *UpdateOptions) error {
                c.Endpoints[docker.DockerEndpoint] = dockerEP
                tlsDataToReset[docker.DockerEndpoint] = dockerTLS
        }
-       if len(o.Kubernetes) != 0 {
-               logrus.Warn("kubernetes orchestrator is deprecated")
-       }
        if err := validateEndpointsAndOrchestrator(c); err != nil {
                return err
        }
```

</details>


```bash
docker context update --kubernetes host=tcp://foo:1234 bla
Flag --kubernetes has been deprecated, option will be ignored
bla
Successfully updated context "bla"
```